config.py.dist

```diff
@@ -10,6 +10,14 @@ BASE_DIR = os.path.abspath(os.path.dirname(__file__))
 # Secret key, CHANGE THIS
 SECRET_KEY = "secret"
```

Comment:
I'm *pretty* sure you can get rid of `SECRET_KEY` because its value isn't used anywhere in the app and other config options have been added to implement session encryption.